linux/mcompile/keymap/mcompile.cpp

```diff
@@ -70,6 +47,16 @@ std::vector<KMX_DeadkeyMapping> KMX_FDeadkeys;  // I4353
   run(argc, argv);
 }
 
+
+/**
+ * @brief  start of mcompile; load, convert and save keyboard
+ * @param  argc    number of commandline arguments
+ * @param  argv     pointer to commandline arguments: executable, inputfile, outputfile
+ * @param  argv_gdk pointer to (commandline arguments)
```

Comment:
There's no `argv_gdk` parameter...

```suggestion
 * @param  argv     pointer to commandline arguments: executable, inputfile, outputfile
```